Update conf-client-mgmt-pf-task.adoc (#276)
mulesoft/docs-access-management: modules/ROOT/pages/conf-client-mgmt-pf-task.adoc

include::_attributes.adoc[]
endif::[]

> Anypoint Platform supports only PingFederate versions 6 through 8. For version 9 and plus, we recommend setting up the Ping Federate client provider as an OIDC provider following xref:configure-client-management-openid-task.adoc[Configure OpenID Connect Client Management]

Reviewer: ```suggestion
Anypoint Platform supports only PingFederate versions 6 through 8. For versions 9 or later, you can configure the PingFederate client provider as an OIDC provider by following the instructions in xref:configure-client-management-openid-task.adoc[Configure OpenID Connect Client Management].
```
